admin/sql/create_models.sql

```diff
@@ -19,4 +19,96 @@ INSERT INTO model (model, model_version, date, status) VALUES ('timbre', 'v2.1_b
 INSERT INTO model (model, model_version, date, status) VALUES ('tonal_atonal', 'v2.1_beta1', now(), 'show');
 INSERT INTO model (model, model_version, date, status) VALUES ('voice_instrumental', 'v2.1_beta1', now(), 'show');
 
+UPDATE model SET class_mapping = '{
```

Comment:
Is there a reason why we're not inserting mappings for `gender`, `voice_instrumental`, `timbre` and `ismir04_rhythm`? Would that json be trivial? I think it might be a good idea to make the `class_mapping` column NOT NULL and insert mappings regardless?

Reply:
Then we could also just put this inside the insert statement which might look a bit cleaner?

Reply:
The original idea was to turn class codes -> real text. In my view, the ones that you said as examples already have names that are nicely readable.
In the future, when we release models built from the dataset editor, I expect that we'll name the classes as nicely readable words (there's no reason that we have to use specific short codes). This means that I expect that we won't need a mapping for any models going forward from here, it's just a quick measure for the existing models that we have.

Reply:
Alright, makes sense.